docs/examples/scheduled_execution.xml

```diff
@@ -18,15 +18,15 @@
 		<!-- Clock variables -->
 		<!-- Periodic input clock -->
 		<Clock name="10msClock"      valueReference="5" causality="input"  intervalVariability="constant"
-		priority="1" intervalCounter="10" resolution="1000"/>
+		priority="1" supportsFraction="true" intervalDecimal="0.01" intervalCounter="10" resolution="1000"/>
```

Comment:
@andreas-junghanns
* isn't intervalCounter="10" resolution="1000" equivalent to intervalDecimal="0.01"? So why change it? ... why leave both options in the standard?
* supportsFraction="true" means that "...  fmi3GetXXXFraction and fmi3SetIntervalFraction are allowed to be called ..." - I don't see a need to call it for a constant clock. Do you?

Reply:
* yes, it is the same, but we made intervalDecimal required to help simple importers to ignore the fractions mechanism.
* indeed, we should add: ... as defined in Table 5.

Reply:
We need a new PR as this one was already merged. Will you (@MBlesken) provide one?